shesha-core/src/Shesha.Application/Notifications/Configuration/NotificationSettings.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Shesha.Configuration;
+using Shesha.Domain;
+using Shesha.Settings;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Shesha.Notifications.Configuration
+{
+    /// <summary>
+    /// Enter preferred channels per priority.
+    /// </summary>
+    public class NotificationSettings
+    {
+        /// <summary>
+        /// Low priority notifications
+        /// </summary>
+        public List<NotificationChannelConfig> Low { get; set; }
```

Comment:
1. Notifications Channels setting doesn't work. Try to changes settings, save and refresh refresh page. Maybe it's a combination of storage mechanism and form configuration. 
2. I checked, you changed storage of entities directly in the settings to guids usage. **NotificationChannelConfig** is a configuration item and it's identified by a pair **module+name**, so it's more correct to save module and name as JSON instead of guid. Guid is pointing to a concrete version of configuration item, current implementation may lead to usage of outdated configurations, or as an alternative you'll have to search latest/live version using non-standard approach (fetch by id, analyze module+name, search for the latest/live version), in any case it will work less stable